backend/app/utils/streaming_jsonl_extractor.py

```
@@ -0,0 +1,184 @@
+    def _is_json_line_start(self, line: str) -> bool:
+        """检查是否是JSON行的开始"""
+        if not line:
+            return False
+        
+        # 简单检查是否以{开始
+        if line.startswith('{'):
+            try:
+                # 尝试部分解析，看是否包含JSONL的基本字段
+                if '"type"' in line or '"t"' in line:
+                    return True
+            except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)